redisinsight/ui/src/components/base/layout/loading-content/LoadingContent.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import {
+  StyledLoadingContent,
+  LoadingContentProps,
+  SingleLine,
+  SingleLineBackground,
+} from './loading-content.styles'
+
+const LoadingContent = ({
+  className,
+  lines = 3,
+  ...rest
+}: LoadingContentProps) => {
+  const classes = classNames('RI-loading-content', className)
+  const lineElements = []
+
+  for (let i = 0; i < lines; i++) {
+    lineElements.push(
+      <SingleLine key={i}>
+        <SingleLineBackground />
+      </SingleLine>,
+    )
+  }
+
+  return (
+    <StyledLoadingContent className={classes} {...rest}>
+      {lineElements}
```

Comment:
an alternative approach keeping jsx on the bottom

```suggestion
{Array(lines).fill().map((_, i) => (
    <SingleLine key={i}>
        <SingleLineBackground />
    </SingleLine>,
))}
```